src/Forms/HTMLEditor/HTMLEditorField.php

```diff
@@ -28,14 +28,6 @@ class HTMLEditorField extends TextareaField
 
     protected $schemaComponent = 'HtmlEditorField';
 
-    /**
-     * Use TinyMCE's GZIP compressor
-     *
-     * @config
-     * @var bool
-     */
-    private static $use_gzip = true;
```

Comment:
I think removing this line is OK as there's no way to access it directly and setting a config flag for a value that doesn't exist doesn't cause an error.